### PR TITLE
43418 add alert section in header

### PR DESCRIPTION
**Pull request by Bibek-Regmi**
## Description
Allow users the ability to display system wide alerts in application header

## Motivation and Context
Jira Issue #43418. -https://cm-jira.usa.gov/browse/IAE-43418

## Type of Change (Select One and Apply Github Label)

- [ ] Bug fix (non-breaking change which fixes an issue) -> Apply bugfix label
- [x] New feature (non-breaking change which adds functionality) -> Apply enhancement label
- [ ] Breaking change (fix or feature that would cause existing functionality to change) -> Apply breaking label

## Which browsers have you tested?
- [ ] Internet Explorer 11
- [ ] Edge
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

## Checklist:

- [ ] I have read the (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md] document.
- [ ] My code passes the automated linter.
- [ ] This code has been reviewed by another team member and passes the reviewer checklist found in (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md]
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code is 508 compliant as tested by AMP and JAWS
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/packages/layouts/src/lib/feature/header/header.component.html [hunk 1]
```diff
@@ -1,89 +1,54 @@
...
-        <fa-icon
-          [icon]="['sds', 'bars']"
-          [classes]="['icon-bars']"
-          transform="shrink-6"
-        ></fa-icon>
-        <fa-layers-counter
-          *ngIf="hasCounter()"
-          [classes]="['icon-layers-counter']"
-        ></fa-layers-counter>
+        <fa-icon [icon]="['fas', 'square']" [classes]="['text-primary']"></fa-icon>
```
Comment (yerramshilpa) [inline libs/packages/layouts/src/lib/feature/header/header.component.html:19]: Relace fa-icon with sds-icon.

File: libs/packages/layouts/src/lib/feature/system-alert/system-alert.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { SdsIconModule } from '@gsa-sam/components';
+
+import { SdsSystemAlertComponent } from './system-alert.component';
+
+fdescribe('SystemAlertComponent', () => {
+  let component: SdsSystemAlertComponent;
```
Comment (yerramshilpa) [inline libs/packages/layouts/src/lib/feature/system-alert/system-alert.component.spec.ts:8]: remove force describe

Comment (Bibek-Regmi) [reply libs/packages/layouts/src/lib/feature/system-alert/system-alert.component.spec.ts:8]: Good catch! Removed

File: libs/packages/layouts/src/lib/feature/header/header.component.html (hunk 1 above)
Comment (Bibek-Regmi) [reply libs/packages/layouts/src/lib/feature/header/header.component.html:19]: Keeping for now as there is no sds icon replacement for square icon

**Review by yerramshilpa — APPROVED** [review]

**Pull request merged**